programmers/난이도별/level02. 방금그곡/sangmandu.py

```diff
@@ -0,0 +1,22 @@
+'''
+https://programmers.co.kr/learn/courses/30/lessons/17683
+방금그곡
+replace로 대문자#을 소문자로 변경.
+'''
+
+def solution(m, musicinfos):
+    m = m.replace('C#', 'c').replace('D#','d').replace('F#','f').replace('G#','g').replace('A#','a')
```

Comment:
```suggestion
    m = m.replace('C#', 'c')
        .replace('D#','d')
        .replace('F#','f')
        .replace('G#','g')
        .replace('A#','a')
```
이런식으로 표현하면 가독성에 더 좋을 것 같아요!